gradle/libs.versions.toml

```
@@ -10,6 +10,14 @@ kotlin-reflect = { module = "org.jetbrains.kotlin:kotlin-stdlib-reflect", versio
 kotest-junit5-jvm = { module = "io.kotest:kotest-runner-junit5-jvm", version.ref = "kotest" }
 kotest-assertions-core-jvm = { module = "io.kotest:kotest-assertions-core-jvm", version.ref = "kotest" }
 mockito-core = { module = "org.mockito:mockito-core", version.ref = "mockito" }
+jcabi-github = { module = "com.jcabi:jcabi-github", version = "1.2.0" }
```

These entries can be simplified:

```suggestion
jcabi-github = "com.jcabi:jcabi-github:1.2.0"
```

The syntax with the TOML object makes sense for shared versions, see next comment.